records_mover/records/prep.py

```diff
@@ -55,11 +56,11 @@ def prep_table_for_load(self,
                                                            self.tbl.table_name)
             if (how_to_prep == ExistingTableHandling.TRUNCATE_AND_OVERWRITE):
                 logger.info("Truncating...")
-                db.execute(f"TRUNCATE TABLE {schema_and_table}")
+                db.execute(text(f"TRUNCATE TABLE {schema_and_table}"))
```

Comment:
also im confused about using the `db` object to do operations, shouldn't these be using the connection instead?

Reply:
I think I might have to do a rewrite of the whole driver interface to get this working; this is going to be a pain.

Reply:
https://bluelabsio.atlassian.net/jira/software/projects/RM/boards/3?selectedIssue=RM-97
ticket here, I'm going to address this in a future PR and not this one if that's alright @Brunope

Reply:
Okay, this is now resolved. All calls are to either db_engine or db_conn depending on what the action is. We've completely separated out these interfaces now.